src/driver_rigel.cpp

```diff
@@ -138,12 +138,22 @@ class PropertyDriverRigel : public IPropertyDriver {
       case CAP_PROP_LEAP_LEDS:
         *max_val = 1;
         break;
+      case CAP_PROP_LEAP_PULSEWIDTHMODULATION: {
+        vector<uint8_t> buf(sizeof(LEAP_DEVCONFIG));

```

Comment:
You could just declare a LEAP_DEVCONFIG instead of creating dynamic vector

Reply:
You're right, that is neater. I was following the convention I'd seen in the rest of the codebase, but I prefer the way you've suggested and changed it.